src/main/java/com/a2/backend/service/impl/ProjectServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.a2.backend.service.impl;
+
+import com.a2.backend.entity.Project;
+import com.a2.backend.exception.ProjectWithThatTitleExistsException;
+import com.a2.backend.model.ProjectCreateDTO;
+import com.a2.backend.repository.ProjectRepository;
+import com.a2.backend.service.ProjectService;
+import lombok.val;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProjectServiceImpl implements ProjectService {
+
+    private final ProjectRepository projectRepository;
+
+    public ProjectServiceImpl(ProjectRepository projectRepository) {
+        this.projectRepository = projectRepository;
+    }
+
+    @Override
+    public Project createProject(ProjectCreateDTO projectCreateDTO) {
+        val existingProjectWithTitle= projectRepository.findByTitle(projectCreateDTO.getTitle());
```

Comment:
Acá por lo que entinedo de los UATS no se debería chequear también que sea del mismo owner el proyecto? Pregunto por la parte que dice que el title es unique "para el mismo usuario".

Reply:
Si, pero como hoy en dia no tenemos users, el proyecto no tiene owner. Deberiamos agregarlo a partir del proximo sprint.

Dejaria un:
`// TODO: verify owner is user that created the project`